From 984263b474387ca4c688f2c830fb3dea16da3ff3 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sun, 29 Jan 2012 16:22:51 -0500
Subject: [PATCH] autogen.sh: Fix check for libtool (we only need libtoolize)

A pre-built libtool copy may not be installed on all systems; all
we need is libtoolize.
---
 autogen.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 426e875..d08b00a 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -26,7 +26,7 @@ touch ChangeLog
 }
 
 (grep "^AM_PROG_LIBTOOL" configure.ac >/dev/null) && {
-  (libtool --version) < /dev/null > /dev/null 2>&1 || {
+  (libtoolize --version) < /dev/null > /dev/null 2>&1 || {
     echo
     echo "**Error**: You must have libtool installed."
     echo "You can get it from: ftp://ftp.gnu.org/pub/gnu/"
-- 
1.7.6.5

